### ISSUE TITLE
[BUG] a11y warning message when clicking user name is not readable in dark mode.

### ISSUE THREAD

**Issue by illume** [Azure/aks-desktop#335]
## Bug Description

a11y warning message when clicking user name is not readable in dark mode.

## Steps to Reproduce

Dark mode
Click your name in the side bar
As it loads there's an error message that has bad contrast.

AKS desktop 0.2.0

**Comment by illume** [comment]
This was fixed by @tejhan at some point a while ago.
